automation/startflexdemo.sh

```diff
@@ -0,0 +1,277 @@
+      httpLogLevel: "$LOG_LEVEL"
+      persistenceLogTarget: "$LOG_TARGET"
+      persistenceLogLevel: "$LOG_LEVEL"
+      persistenceDurationLogTarget: "$LOG_TARGET"
+      persistenceDurationLogLevel: "$LOG_LEVEL"
+      ldapStatsLogTarget: "$LOG_TARGET"
+      ldapStatsLogLevel: "$LOG_LEVEL"
+      scriptLogTarget: "$LOG_TARGET"
+      scriptLogLevel: "$LOG_LEVEL"
+      auditStatsLogTarget: "$LOG_TARGET"
+      auditStatsLogLevel: "$LOG_LEVEL"
+  casa:
+    # -- App loggers can be configured to define where the logs will be redirected to and the level of each in which it should be displayed.
+    appLoggers:
+      casaLogTarget: "$LOG_TARGET"
+      casaLogLevel: "$LOG_LEVEL"
+      timerLogTarget: "$LOG_TARGET"
+      timerLogLevel: "$LOG_LEVEL"
+    enabled: true
+    ingress:
+      casaEnabled: true    
+  config-api:
+    enabled: true
+    appLoggers:
+      configApiLogTarget: "$LOG_TARGET"
+      configApiLogLevel: "$LOG_LEVEL"
+    adminUiAppLoggers:
+      # -- config-api admin-ui plugin log level
+      adminUiLogTarget: "$LOG_TARGET"
+      adminUiLogLevel: "$LOG_LEVEL"
+      adminUiAuditLogTarget: "$LOG_TARGET"
+      adminUiAuditLogLevel: "$LOG_LEVEL"
+  fido2:
+    enabled: true
+    ingress:
+      fido2ConfigEnabled: true
+    appLoggers:
+      fido2LogTarget: "$LOG_TARGET"
+      fido2LogLevel: "$LOG_LEVEL"
+      persistenceLogTarget: "$LOG_TARGET"
+      persistenceLogLevel: "$LOG_LEVEL"
+  scim:
+    enabled: true
+    ingress:
+      scimConfigEnabled: true
+      scimEnabled: true
+    appLoggers:
+      scimLogTarget: "$LOG_TARGET"
+      scimLogLevel: "$LOG_LEVEL"
+      persistenceLogTarget: "$LOG_TARGET"
+      persistenceLogLevel: "$LOG_LEVEL"
+      persistenceDurationLogTarget: "$LOG_TARGET"
+      persistenceDurationLogLevel: "$LOG_LEVEL"
+      ldapStatsLogTarget: "$LOG_TARGET"
+      ldapStatsLogLevel: "$LOG_LEVEL"
+      scriptLogTarget: "$LOG_TARGET"
+      scriptLogLevel: "$LOG_LEVEL"
+  fqdn: $GLUU_FQDN
+  isFqdnRegistered: false
+  lbIp: $EXT_IP
+  opendj:
+    # -- Boolean flag to enable/disable the OpenDJ  chart.
+    enabled: $ENABLE_LDAP
+# -- Nginx ingress definitions chart
+nginx-ingress:
+  ingress:
+    openidConfigEnabled: true
+    uma2ConfigEnabled: true
+    webfingerEnabled: true
+    webdiscoveryEnabled: true
+    scimConfigEnabled: true
+    scimEnabled: true
+    configApiEnabled: true
+    u2fConfigEnabled: true
+    fido2ConfigEnabled: true
+    authServerEnabled: true
+    path: /
+    hosts:
+    - $GLUU_FQDN
+    # -- Secrets holding HTTPS CA cert and key.
+    tls:
+    - secretName: tls-certificate
+      hosts:
+      - $GLUU_FQDN
+auth-server:
+  livenessProbe:
+    # https://github.com/JanssenProject/docker-jans-auth-server/blob/master/scripts/healthcheck.py
+    exec:
+      command:
+        - python3
+        - /app/scripts/healthcheck.py
+    # Setting for testing purposes only. Under optimal resources the app should be up in 30-60 secs
```

Comment:
Can you remove all the default values so that the override is a little shorter

Reply:
Removed in both flex and jans